### PR TITLE
ci: add link checking

### LINKED ISSUE #7: Broken links
@maelle: can we run a checker for broken links here, similarly to elsewhere?

### PR DESCRIPTION
**Pull request by maelle**
Fix #7

### CODE REVIEW

File: blog/dbi-4-1.qmd [hunk 1]
```diff
@@ -3,6 +3,8 @@ author: "Kirill Müller"
 date: "2024-04-02"
 title: "The future of DBI (1/*)"
 description: "Summarizing the progress of 2022 and 2023"
+aliases:
+  - posts/dbi-4-1.html
```
Comment (maelle) [inline blog/dbi-4-1.qmd]: I'll have to test the redirect

**Comment by krlmlr** [comment]
This is now spinning.

**Comment by maelle** [comment]
Works on my machine :joy_cat:

**Comment by maelle** [comment]
many links are broken because of the new structure, so I'll add the r-dbi website to the ignore list as we did for cynkrablog.

**Pull request merged**